docker-sync.yml

```diff
@@ -0,0 +1,17 @@
+version: "2"
```

Comment:
Correct: http://docker-sync.io/

Reply:
Asking again, will this interfere with people using the setup we already have?

Reply:
It worked fine for me, but I keep ruby outside a container (easier to start the IntelliJ debugger that way)

Reply:
I don't think that we should add this. docker-sync is useful for who uses Docker on Mac, but it sometimes stops syncing. Maybe writing down in wiki page; https://github.com/Safecast/safecastapi/wiki/Dev:-Setup-on-Linux-and-OS-X-using-Docker ?

Reply:
@eitoball Sure, I'll update in wiki

Reply:
Oh... What I wanted to say is that `we should NOT add this file,  and this content should go into wiki` because you need to use another binary, and some people like running rails outside docker container or running container on Linux .

Reply:
But what to do with these like me who is using docker on his Mac?